python/taichi/lang/__init__.py

```diff
@@ -251,6 +251,8 @@ def supported_archs():
     if ti.core.with_opengl():
         archs.append(opengl)
     wanted_archs = os.environ.get('TI_WANTED_ARCHS', '').split(',')
+    # Note, ''.split(',') gives you [''], which is not an empty array.
+    wanted_archs = list(filter(lambda x: x != '', wanted_archs))
```

Comment:
Nice catch!,maybe simply len instead of lambda x: x != '' works?

Reply:
I think both will work and `x != ''` seems clearer to me. Thanks for the prompt review - I'll merge once CI passes.